getting-started/views/view-helpers.md

```diff
@@ -0,0 +1,198 @@
+# Views Helpers
+
+Views helpers are available from the [Jasper::Helpers](https://github.com/amberframework/jasper-helpers) library.
+
+## Links
+A `link_to` helper is available:
+
+```slim
+== link_to "Home", "/"
```

Comment:
Can you change all the `==` to a `=`?  You only use a double `==` when you don't want the html escaped.